global_translations.R

```diff
@@ -38,3 +38,27 @@ complete_trans_df <- subset(final_df, untranslated_count == 0, select = -c(fuzzy
 
 # Untranslated Packages with respective languages associated
 complete_untrans_df <- subset(final_df, translated_count == 0, select = -c(fuzzy_count))
+
+# R Language Translation Team Contact details
+library(rvest)
+library(stringr)
+
+translation_teams <- read_html("https://developer.r-project.org/TranslationTeams.html") |>
+  html_node("table") |>
+  html_table()
+
+members_contact <- translation_teams$Contact |>
+  gsub(pattern = ".*<", replacement = "") |>
+  gsub(pattern = ">", replacement = "") |>
+  gsub(pattern = ",.*", replacement = "") |> 
```

Comment:
I don't see any commas on the current page, what's this line for?

Reply:
Actually this was from earlier code. Where the name written with a comma and no email address. When page was not updated earlier.
Will remove that.